include/cista/targets/file.h

```diff
@@ -260,33 +260,35 @@ struct file {
   template <typename T>
   void write(std::size_t const pos, T const& val) {
     verify(!std::fseek(f_, static_cast<long>(pos), SEEK_SET), "seek error");
-    verify(std::fwrite(reinterpret_cast<unsigned char const*>(&val), 1,
+    verify(std::fwrite(reinterpret_cast<std::uint8_t const*>(&val), 1U,
                        serialized_size<T>(), f_) == serialized_size<T>(),
            "write error");
   }
 
   offset_t write(void const* ptr, std::size_t const size,
                  std::size_t alignment) {
     auto curr_offset = size_;
-    if (alignment != 0 && alignment != 1) {
+    long seek_offset = 0;
+    int seek_whence = SEEK_END;
```

Comment:
Can you please follow the "aaa style": https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/

Reply:
- line 271, if `auto seek_offset = 0` would be used the type will be in fact `int` but `long` is expected.
- line 272, `auto` would be indeed the right way if the right side value would be a typed value but since SEEK_END is a macro I have doubts and thus I preferred an explicit `int` as type. But if you think `auto` will be more appropriate, I can change it.

Reply:
As stated in the article, this would be the style.

```
auto x = long long{ 42 };            // error
auto x = int64_t{ 42 };              // ok, better 
auto x = 42LL;                       // ok, better 
```

So it's possible to use the aaa style, even if the type is not int.

Reply:
As you prefer that but IMO aaa style for the initialization with 0 or a constant is over-hyped.